autoprofile/internal/frame_filter.go

```diff
@@ -8,33 +8,20 @@ import (
 )
 
 var (
-	IncludeSensorFrames = true
-	sensorPath          = filepath.Join("github.com", "instana", "go-sensor")
+	IncludeProfilerFrames = false
+	autoprofilePath       = filepath.Join("instana", "go-sensor", "autoprofile")
```

Comment:
Did I read this correctly, with this we want to exclude only the profiler frames, but not the whole sensor?

Reply:
Also you might consider making this path more strict and keep the `github.com` prefix there just in case this library is used in the code that has `instana/go-sensor/autoprofile` path for some reason.

Reply:
Yes, we will exclude only profiler stacks for now. We do not automatically instrument other parts of sensor, so there is not much to exclude.
Regarding the path being more strict, good point, I'll add the github.com back, it was removed for local testing.